web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -645,15 +652,15 @@ handlers:
+      tags: 'drupal webform campsite'
       priority: normal
       status: new
       group_id: '18716157058327'
       assignee_id: ''
       type: incident
       collaborators: ''
-      custom_fields: "6353388345367: 'report_campsite'\r\n14510509580823: 'true'\r\n5581480390679: ['[webform_submission:values:report_location:location_lat]', 'report_location:location_lat']\r\n5581490332439: ['[webform_submission:values:report_location:location_lon]', 'report_location:location_lon']\r\n13407901552407: ['[webform_submission:values:report_location:location_x]', 'report_location:location_x']\r\n13407918835095: ['[webform_submission:values:report_location:location_y]', 'report_location:location_y']\r\n1500012743961: ['[webform_submission:values:report_location:location_address]', 'report_location:location_address']\r\n1500013095781: '[webform_submission:uuid]'\r\n9205221816983: ['[webform_submission:values:computed_property_type]', 'property_type']\r\n14971313641239: ['[webform_submission:values:report_vehicle:color:raw]', 'report_vehicle:color']\r\n5873310799383: ['[webform_submission:values:report_vehicle:license_plate_number]', 'report_vehicle:license_plate_number']\r\n18716068517783: ['[webform_submission:values:report_vehicle:license_plate_state:raw]', 'report_vehicle:license_plate_state']\r\n5873293014679: ['[webform_submission:values:report_vehicle:make:raw]', 'report_vehicle:make']\r\n14971350075799: ['[webform_submission:values:report_vehicle:type:raw]', 'report_vehicle:type']"
+      custom_fields: "6353388345367: 'report_campsite'\r\n14510509580823: 'true'\r\n5581480390679: ['[webform_submission:values:report_location:location_lat]', 'report_location:location_lat']\r\n5581490332439: ['[webform_submission:values:report_location:location_lon]', 'report_location:location_lon']\r\n13407901552407: ['[webform_submission:values:report_location:location_x]', 'report_location:location_x']\r\n13407918835095: ['[webform_submission:values:report_location:location_y]', 'report_location:location_y']\r\n1500012743961: ['[webform_submission:values:report_location:location_address]', 'report_location:location_address']\r\n1500013095781: '[webform_submission:uuid]'\r\n9205221816983: ['[webform_submission:values:computed_property_type]', 'property_type']\r\n14971313641239: ['[webform_submission:values:report_vehicle:color:raw]', 'report_vehicle:color']\r\n5873310799383: ['[webform_submission:values:report_vehicle:license_plate_number]', 'report_vehicle:license_plate_number']\r\n18716068517783: ['[webform_submission:values:report_vehicle:license_plate_state:raw]', 'report_vehicle:license_plate_state']\r\n5873293014679: ['[webform_submission:values:report_vehicle:make:raw]', 'report_vehicle:make']\r\n14971350075799: ['[webform_submission:values:report_vehicle:type:raw]', 'report_vehicle:type']\r\n8692493739543: '[webform_submission:values:report_ticket_id]'"
```

Comment:
Stores the related campsite ticket ID in the Abandoned Autos ticket, in the Related Ticket ID custom field in Zendesk. This custom field already exists but is not being added to the campsite form, so this does not change the schema. This handler is currently disabled.